guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<ul><li><code><a href="https://developers.arcgis.com/python/guide/" target="_blank">Working with the Spatially Enabled DataFrame</a></code></li><li class="ql-indent-1"><a href="https://developers.arcgis.com/python/guide/introduction-to-the-spatially-enabled-dataframe/" target="_blank">Introduction to the Spatially Enabled DataFrame</a></li><li class="ql-indent-1"><s>Updates the entire guide</s></li></ul><p>(we can remove the last bullet as we are only updating one of 6 parts in this release)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>added phrase stating updating part of guide</p>